### ISSUE TITLE
]LINKed classes do not preserve stops

### ISSUE THREAD

**Issue by mbaas2** [Dyalog/link#228]
**Describe the bug**
Stops set in a fn within class are removed after closing editor. (They are kept in simple fns)

**To Reproduce**
Steps to reproduce the behaviour:
1. ]LINK.CREATE # {anyPath}
2. )ed ○foo
2. Paste the following:
 ````
:Class foo
    ∇ R←a plus b
      :Access public shared
      R←a+b
    ∇
:EndClass
````
3. set a stop point in `plus`
4. fix the fn
5. edit it again - stop has gone!

**Expected behaviour**
Stop points should be kept after editing

**Desktop (please complete the following information):**
Dyalog  18.1.40069 64-bit Unicode, BuildID 6f643263            
 OS      Windows 10 or Windows Server 2016 (10.0.19041) 64-bit  
 SALT    2.9, Link: 2.1.0-beta42                                
 UCMD    2.5                                                    
 .NET    .NET Core 3.1.12                                       
 WS      18.1

**Comment by nicolas-dyalog** [comment]
Requires implementing RFE Mantis 18942
Or at the very least fix bug Mantis 18943

**Comment by mkromberg** [comment]
Fixed in 18.2.44492  (Mantis 19293).
